modules/cluster-logging-json-log-forwarding.adoc

```diff
@@ -31,7 +22,7 @@ pipelines:
   parse: json
 ----
 
-When you enable parsing JSON logs by using `parse: json`, the CR copies the JSON-structured log entry in a `structured` field, as shown in the following example. This does not modify the original `message` field.
+When you enable parsing JSON logs by using `parse: json`, the CR copies the JSON-structured log entry in a `structured` field, as shown in the following example.
```

Comment:
Up to you but I think ending this with a colon might make a stronger connection between this sentence and the example itself:
```suggestion
When you enable parsing JSON logs by using `parse: json`, the CR copies the JSON-structured log entry in a `structured` field, as shown in the following example:
```

This would be out of scope but if you choose to use a colon, the same can be done for line 14 above this.